tools/analyzer_plugin/tool/add_over_react_override.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:io';
+
+import 'package:logging/logging.dart';
+import 'package:path/path.dart' as p;
+import 'package:yaml/yaml.dart';
+
+import 'logging.dart';
+
+final logger = Logger('add_over_react_override');
+
+/// Script to add a standard over_react dependency override.
+///
+/// This is primarily used for CI to check against the most recent, unpublished changes to over_react.
+void main(List<String> args) {
+  initLogging(verbose: args.contains('--verbose'));
+
+  final packageRoot = p.canonicalize(p.join(Platform.script.toFilePath(), '..', '..'));
+  final pluginPubspec = File(p.join(packageRoot, 'pubspec.yaml'));
```

Comment:
Isn't this the root pubspec and not the plugin pubspec?

Reply:
No, it's relative from this script, `tool/add_over_react_override.dart`